### PR TITLE
InstructorHomePage: change delete course confirmation message #3118

### LINKED ISSUE #3118: InstructorHomePage: change delete course confirmation message
Are you sure you want to delete the course: .... ?
This operation will delete all students<b>, feedback sessions</b> and evaluations in this course.
All instructors of this course will not be able to access it hereafter as well.

### PR DESCRIPTION
**Pull request by Nilaksha**
Fixes #3118

### CODE REVIEW

File: .settings/com.google.appengine.eclipse.core.prefs [hunk 1]
```diff
@@ -3,6 +3,7 @@ filesCopiedToWebInfLib=appengine-api-1.0-sdk-1.9.4.jar|appengine-api-labs.jar|ap
 gaeDatanucleusVersion=v1
 gaeDeployDialogSettings=
 gaeIsEclipseDefaultInstPath=true
+gaeLaunchAppInBrowser=true
```
Comment (kanghj) [inline .settings/com.google.appengine.eclipse.core.prefs]: Remove changes unrelated to the issue from the pull request

**Comment by kanghj** [comment]
The branch name is wrong, it should be named as the issue. You need to move your commits to a correctly-named branch and make a new pull request.

**Comment by Nilaksha** [comment]
I'm confused. I followed the steps in Fixing Issues under TEAMMATES
Development Process. If the branch name is wrong, what should be the
correct branch name? Please help me out.

Nilaksha Perera,
Undergraduate,
Department of Computer Science and Engineering,
University of Moratuwa,
Sri Lanka.

**Comment by kanghj** [comment]
Under Fixing Issues, step 6: Start a new branch named Issue{IssueNumber}

For this issue, it should be named Issue3118.

**Comment by Nilaksha** [comment]
Thanks a lot. Now I know what I should do. I'll make a new pull request.

**Pull request closed without merging**